cassandane/Cassandane/Instance.pm

```diff
@@ -1408,12 +1408,18 @@ sub _detect_core_program
     my $lines = 0;
     my $prog;
 
+    my $bindir_pattern = qr{
+        \/
+        (?:bin|sbin|libexec)
+        \/
+    }x;
+
```

Comment:
It'd be nice if the commit message that added this explain why this was changing (as it seems unrelated to core patterns)

Reply:
done, how's that?

Reply:
Thanks